src/kayenta/stages/kayentaStage/kayentaStage.html

```diff
@@ -346,7 +346,7 @@
               </tr>
               </tbody>
               <tfoot>
-              <tr ng-if="!kayentaCanaryStageCtrl.stage.deployments.serverGroupPairs.length">
+              <tr>
```

Comment:
Orca doesn't support multiple pairs...the model supports it, but the other pairs won't be canaried.

Reply:
I see — that was not the impression I had. Will yank this out for now and we'll knock out multi-pair support in orca before re-adding.

Reply:
I must have misread the code; looks like i have Orca work to do.

Reply:
@danielpeach this is pulled out now fyi